packages/autocomplete-js/src/utils/pickBy.ts

```diff
@@ -0,0 +1,15 @@
+export function pickBy<TValue = unknown>(
+  obj: Record<string, TValue>,
+  predicate: (value: { key: string; value: TValue }) => boolean
+) {
+  return Object.entries(obj).reduce<Record<string, TValue>>(
```

Comment:
Type inference should be working here.

```suggestion
  return Object.entries(obj).reduce(
```

Reply:
Unfortunately it doesn't because the return type is inferred from the initial value (here, `{}`). It's either this or casting the initial value.